reana/reana_dev/git.py

```diff
@@ -997,4 +997,46 @@ def _git_create_pr(comp):
             _git_create_pr(component)
 
 
+@click.option(
+    "--component",
+    "-c",
+    required=True,
+    multiple=True,
+    help="Which components? [name|CLUSTER]",
+)
+@git_commands.command(name="git-tag")
+def git_tag(component):  # noqa: D301
+    """Create the corresponding git tag for components with release commits.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components;
+                         * (5) special value 'CLIENT' that will expand to
+                               cover all REANA client components;
+                         * (6) special value 'DEMO' that will expand
+                               to include several runable REANA demo examples;
+                         * (7) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :type component: str
+    """
+    components = select_components(component)
+    for component in components:
+        if not is_last_commit_release_commit(component):
+            click.secho(
+                "The last commit is not a release commit. Please use `reana-dev git-create-release-commit`.",
```

Comment:
Works well :+1: 

Cosmetic comment: in the output message

```
$ reana-dev git-tag -c r-commons
The last commit is not a release commit. Please use `reana-dev git-create-release-commit`.
```

we could print out also argument, so that users can copy/paste if they wish:

```
The last commit is not a release commit. Please use `reana-dev git-create-release-commit -c r-commons` first.
```